bundles/org.openhab.binding.growatt/src/main/resources/OH-INF/i18n/growatt.properties

```diff
@@ -0,0 +1,241 @@
+# add-on
+
+addon.growatt.name = Growatt Binding
+addon.growatt.description = This is the binding for Growatt solar inverters.
+
+# thing types
+
+thing-type.growatt.bridge.label = Growatt Bridge Thing
+thing-type.growatt.bridge.description = Bridge Thing for Growatt Binding
+thing-type.growatt.inverter.label = Growatt Inverter Thing
+thing-type.growatt.inverter.description = Inverter Thing for Growatt Binding
+thing-type.growatt.inverter.channel.battery-display.label = Battery Display
+thing-type.growatt.inverter.channel.battery-display.description = Battery display code.
+thing-type.growatt.inverter.channel.battery-soc.label = Battery Charge
+thing-type.growatt.inverter.channel.battery-soc.description = Battery state of charge.
+thing-type.growatt.inverter.channel.battery-temperature.label = Battery Temperature
+thing-type.growatt.inverter.channel.battery-temperature.description = Battery temperature.
+thing-type.growatt.inverter.channel.battery-type.label = Battery Type
+thing-type.growatt.inverter.channel.battery-type.description = Type code of the battery.
+thing-type.growatt.inverter.channel.battery-voltage.label = Battery Voltage
+thing-type.growatt.inverter.channel.battery-voltage.description = Battery voltage.
+thing-type.growatt.inverter.channel.charge-current.label = Charge Current
+thing-type.growatt.inverter.channel.charge-current.description = Charge current to battery.
+thing-type.growatt.inverter.channel.charge-power.label = Charge Power
+thing-type.growatt.inverter.channel.charge-power.description = Charge power to battery.
+thing-type.growatt.inverter.channel.constant-power-ok.label = Constant Power OK
+thing-type.growatt.inverter.channel.constant-power-ok.description = Constant power OK code.
+thing-type.growatt.inverter.channel.discharge-energy-today.label = Energy from Battery Today
```

Comment:
Labels should be capitalized, short (< 23 chars) and 2-3 words. 
Most are perfect, but please check others .

Obviously, they need to be fixed in the XML and this file will be updated by i18n plugin. But reviewing these labels here is way easier :-)

Reply:
@lsiepel for the avoidance of doubt can you please advise what you want to change?
- Are you asking me to change the [Title Case](https://en.wikipedia.org/wiki/Title_case) of prepositions like 'from' and 'to'? .. personally I follow my school English teacher's rule as described in the Wiki _"When using title case, all words are capitalized, **except** for minor words (typically [articles](https://en.wikipedia.org/wiki/Article_(grammar)), short [prepositions](https://en.wikipedia.org/wiki/Preposition), and some [conjunctions](https://en.wikipedia.org/wiki/Conjunction_(grammar))) that are not the first or last word of the title."_
- Or are you asking me to shorten "Energy from Battery Today" to three words? Note that the preposition (even if not capitalise) is essential to differentiate "Energy from Battery Today" vs "Energy to Battery Today" etc.

Reply:
PS https://content.cam.ac.uk/style-guide/#C :)

Reply:
I completely trust you and your English teacher to be right in general about the capitalization. In general, for UI elements, we sometimes tend to deviate from those rules as space is limited, UI's have high information density and can benefit from clean/consistency.

I'm not sure how strict we (openHAB) take these guidelines. So i tend to leave it as is. Maybe another @openhab/add-ons-maintainers can comment to be sure. For what it's worth, there are some labels like "Import Energy to Charge Battery Total" are a bit more problematic regarding the length. On the other side, i fully understand that it is hard to shorten it without loosing a proper funcitonal meaning.

Reply:
See: https://github.com/openhab/static-code-analysis/pull/360

Reply:
In today's commit I have shortened quite a few of the label texts.
I think it is not possible to shorten them any more.